docs/contributing.md

```diff
@@ -4,41 +4,44 @@ title: Contributing Guide
 sidebar_label: Contributing Guide
 ---
 
-## Thank you for participating
+## Thank you for participating 
 
 The following is a set of rules for contributing to **Rucio** and its
 packages. Use your best judgment, and feel free to propose changes to this
 document.
 
-If you have questions, you can reach the core development team on our 
-[__Mattermost__](mattermost.md) channel, or send an email to our
-development mailing list [__rucio-dev@cern.ch__](mailto:rucio-dev@cern.ch).
+If you have questions, you can reach the development team on our 
+[__Mattermost__](mattermost.md) channel.
```

Comment:
I would point them to the contact us page instead: https://rucio.cern.ch/documentation/contact_us